README.md

```diff
@@ -44,10 +47,11 @@ $ ./test.sh repo --spec specs/repo/special-item.ts
 $ ./test.sh repo --setup
 ```
 
-### Example Instance Commands
+### Deploy
 
 ```
-$ docker compose --env-file ./example/template.env up --wait
+$ cd deploy
```

Comment:
I may have missed it elsewhere, but I'm not seeing where we guide the user to cp `template.env` to `.env`. Also, as mentioned in my earlier comment, I would still prefer to keep these docs in the `deploy` directory. 

Beyond those two things though, I appreciate the brevity here and which I think encourages users to look more careful at comments in the configuration files.